lib/5.26.2/x86_64-linux/CORE/config.h

```diff
@@ -3773,9 +3773,9 @@
  */
 #define DB_Hash_t	u_int32_t		/**/
 #define DB_Prefix_t	size_t  	/**/
-#define DB_VERSION_MAJOR_CFG	5  	/**/
-#define DB_VERSION_MINOR_CFG	3  	/**/
-#define DB_VERSION_PATCH_CFG	28  	/**/
+#define DB_VERSION_MAJOR_CFG	  	/**/
```

Comment:
if the definition are no longer needed why not completely remove them?

Reply:
This one is  generated by `./configure` command. It is not maintained by us.